pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -146,23 +152,30 @@ def __init__(
         cutoff_mode: str = "abs",
         device_name="lightning.tensor",
     ):
-        self._num_wires = num_wires
-        self._max_bond_dim = max_bond_dim
-        self._method = method
-        self._cutoff = cutoff
-        self._cutoff_mode = cutoff_mode
-        self._c_dtype = c_dtype
-
         if device_name != "lightning.tensor":
             raise DeviceError(f'The device name "{device_name}" is not a valid option.')
 
         if num_wires < 2:
             raise ValueError("Number of wires must be greater than 1.")
 
+        self._num_wires = num_wires
+        self._method = method
```

Comment:
Alright! I see that you followed[ PL's UI](https://github.com/PennyLaneAI/pennylane/blob/69bd9ff3e17f026c4dc9da8737d37ad09fc11e3f/pennylane/devices/default_tensor.py#L374C1-L375C1). I take back my earlier comment in `_serialize,py `. But still defauling `tensor_backend` as mps doesn't make sense for state-vector simulators!

Reply:
Ok I will revert the `backend` to `tensor_backend`